src/lib/modules/client/terminal/xterm-wrapper.ts

```
@@ -67,6 +71,59 @@ export async function createTerminal(options: TerminalOptions): Promise<Terminal
   term.open(options.container);
   fitAddon.fit();
 
+  // Block Cmd/Ctrl+key shortcuts from reaching the PTY (except copy/paste)
+  term.attachCustomKeyEventHandler((e) => {
+    if (e.metaKey || (e.ctrlKey && !e.altKey)) {
+      if (e.key === 'c' || e.key === 'v') return true; // allow copy/paste
```

ESLint

[error] Expected { after 'if' condition.

(curly)